.markdownlint.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Can you please stick with the standard template.
https://github.com/dnsimple/dnsimple-engineering/blob/main/templates/markdownlint.yaml